cassandra/src/cassandra/lwt.clj

```diff
@@ -5,7 +5,9 @@
             [jepsen
              [checker :as checker]
              [client :as client]
-             [generator :as gen]]
+             [generator :as gen]
+             [independent :as independent]]
```

Comment:
Just a question. I'm not sure about it at all but just curious if there is any downside of using `independent` since it seems like partitioning data/test but there might a case where the partitioned data/test are dependent each other ?
So, is it something that brings lower memory footprint but might bring some false positive/negative results ? or the same correctness with slower checking ?
Sorry for rough question.

Reply:
Good question!
Each part that is partitioned by `independent` is checked.
In the test, the first part requests for the primary key `id` = 0, the second part requests for `id` = 1 ... At the last, the checker for linearizability checks each part, so the checker checks the value where `id` = 0 in the first part. That's why each part is independent.

In other words, `independent` repeats small tests. I think it might bring some false-negative results. However, if `independent` isn't used and we try to avoid out of memory, we cannot inject failures because the test time is too short to inject them.